featuretools/entityset/timedelta.py

```diff
@@ -44,48 +44,37 @@ class Timedelta(object):
         "o": "Observations",
         "w": "Weeks",
         "Y": "Years",
-        'mo': "Months"
-    }
-
-    _convert_to_days = {
-        "w": 7
+        "mo": "Months"
     }
 
     _readable_to_unit = {v.lower(): k for k, v in _readable_units.items()}
 
-    def __init__(self, value, unit=None):
+    def __init__(self, value, unit=None, delta_obj=None):
```

Comment:
Should we document the `delta_obj` parameter?

Reply:
I think we should so it's clear that it's more for _check_timedelta() and it's not necessary if a user only wants to use value and units to instantiate.